src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+import Icon from "../Icon/Icon";
+import {
+  Selector,
+  Item,
+  Wrapper,
+  Arrow,
+  AbsoluteWrapper,
+} from "./Dropdown.styles";
+
+export type SelectOption = {
+  label: string;
+  value: string;
+};
+
+export type DropdownProps = {
+  category: SelectOption;
+  options: SelectOption[];
+  onChange: (option: SelectOption) => void;
+};
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  category,
+  options,
+  onChange,
+}) => {
+  const [selectedOption, setSelectedOption] = useState(category);
+
+  function handleChange(newOption: SelectOption) {
+    setSelectedOption(newOption);
+    onChange(selectedOption);
+  }
+
+  return (
+    <Wrapper>
+      <AbsoluteWrapper
+        onClick={(event) => {
+          event.currentTarget.classList.toggle("show");
+        }}
+      >
+        <Selector>
+          <Item
+            className={`${
+              selectedOption === category ? "selected category" : "category"
```

Comment:
No reason to show `'selected category'`, `selectedOption` is required so just show its `label`.